### PR TITLE
Incrementalified.

### PR DESCRIPTION
**Pull request by timja**
Requires hosting permission PR created and merged first

https://github.com/jenkinsci/incrementals-tools

### CODE REVIEW

**Comment by timja** [comment]
Pipeline is failing with:

```
No tool named mvn-default found
```

cc @XiongKezhi

File: .mvn/maven.config [hunk 1]
```diff
@@ -0,0 +1 @@
+-Pconsume-incrementals
```
Comment (uhafner) [inline .mvn/maven.config:1]: Shouldn't it also contain
```
-Pmight-produce-incrementals
```

File: pom.xml [hunk 2]
```diff
@@ -12,12 +12,14 @@
...
   <groupId>io.jenkins.plugins</groupId>
   <artifactId>github-checks</artifactId>
-  <version>0.1.0-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
   <packaging>hpi</packaging>
   <description>Allows users to publish GitHub checks</description>
 
   <properties>
     <java.level>8</java.level>
+    <revision>0.1.0</revision>
```
Comment (uhafner) [inline pom.xml:21]: I'm not sure if it makes sense to publish a beta? Then someone can install it from the experimental update center.
```suggestion
    <revision>0.1.0-beta-1</revision>
```

File: .mvn/maven.config (hunk 1 above)
Comment (timja) [reply .mvn/maven.config:1]:
```suggestion
-Pconsume-incrementals
-Pmight-produce-incrementals
```

File: pom.xml (hunk 2 above)
Comment (timja) [reply pom.xml:21]: can always add the beta in at release time?

**Pull request merged**